python-threatexchange/threatexchange/signal_type/tests/signal_type_test_helper.py

```diff
@@ -0,0 +1,115 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+
+import pytest
+import typing as t
+
+from threatexchange.signal_type.signal_base import (
+    MatchesStr,
+    TextHasher,
+    SignalType,
+)
+
+
+class SignalTypeHashTest:
+    """
+    Helpers to help you quickly test common functionality
+
+    Most functions are designed to be used to be used in a provider fasion
+    to rapidly make simple tests.
```

Comment:
nit: add a little more context on how to use and/or point read to `test_raw_text.py` as an example.

Reply:
I have something stacked on top of this, will add it there.